centreon/src/Core/HostTemplate/Infrastructure/API/AddHostTemplate/AddHostTemplateSaasSchema.json

```diff
@@ -104,6 +104,24 @@
                 "type": "integer"
             }
         },
+        "event_handler_enabled": {
+          "type": [
+            "null",
+            "integer"
+          ],
+          "enum": [
+            null,
+            0,
+            1,
+            2
```

Comment:
Should this value be nullable ? According to the ticket the enabled status should be "Yes"/"No"/"Default"